src/poprox_storage/concepts/manifest.py

```diff
@@ -19,8 +28,15 @@
 
 class ManifestFile(BaseModel):
     """
-    Parses and validates experiment manifest files that have
-    been converted from TOML to Dict[str,Any] (e.g. using tomli)
+    Parses and validates experiment manifest files that have been converted
+    from TOML to Dict[str,Any] (e.g. using tomli).
+
+    Attributes:
+        experiment: Experiment metadata and configuration
+        owner: Experiment team owner
+        users: Specification of user groups participating in the experiment
+        recommenders: Dictionary mapping recommender names to their configurations
+        phases: Sequence of experiment phases
```

Comment:
I was not sure if these would attach to the correct places in the rendered documentation, but I have prototyped adding the docs to the rendered manual, and they do display correctly.

I usually prefer to put attribute documentation with the attribute itself instead of in the class docstring to minimize distance between the documentation and the thing documented, but that's a change we can make later / in future documentation too.